lib/task_bunny/status.ex

```diff
@@ -36,6 +37,51 @@ defmodule TaskBunny.Status do
     end
   end
 
+  @doc ~S"""
+  Returns TaskBunny metrics.
+  """
+  @spec metrics(supervisor :: atom) :: keyword
+  def metrics(supervisor \\ TaskBunny.Supervisor) do
+    case supervisor_alive?(supervisor) do
+      true ->
+        {runners, succeeded, failed, rejected} =
+          supervisor
+          |> get_workers_status()
+          |> Enum.reduce({[], [], [], []}, &get_worker_metrics/2)
+
+        [
+          task_bunny_job_runners_now: {runners, :gauge, "Current amount of TaskBunny runners."},
```

Comment:
Is it the number of the job executed since app started? Or purely the number of job currently running?